src/main/java/com/work/management/entity/Employee.java

```diff
@@ -0,0 +1,43 @@
+package com.work.management.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Builder
+@Entity
+@Table(name = "EMPLOYEES")
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
Add @EqualsAndHashCode otherwise collections like HashMap won't work properly.
Use @Data instead of using these many annotations.